catalyst-gateway/bin/Cargo.toml

```diff
@@ -70,3 +71,10 @@ ulid = { workspace = true, features = ["serde", "uuid"] }
 rust-embed = { workspace = true }
 local-ip-address.workspace = true
 gethostname.workspace = true
+
+hex = "0.4.3"
```

Comment:
I think it is better to move all dependencies definitions to the workspace level.

Reply:
Ok, I'll address this in another PR.